# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1461178837 14400
#      Wed Apr 20 15:00:37 2016 -0400
# Branch stable
# Node ID acaf1e137c5e6f201a6689a99a6568d0dfe9e9ed
# Parent  9860b8183202e7d4b4f57ff8dc74773c70b3798e
Version 4.0.2 released.

* configure.ac: Bump version to 4.0.2.
(OCTAVE_PATCH_VERSION): Bump to 2.
(OCTAVE_RELEASE_DATE): Update.
* liboctave/Makefile.am (liboctave_revision): Increment.
* libinterp/Makefile.am (liboctinterp_revision): Increment.

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -14,31 +14,31 @@ dnl Copyright (C) 1993-2015 John W. Eato
 ### FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
 ### for more details.
 ###
 ### You should have received a copy of the GNU General Public License
 ### along with Octave; see the file COPYING.  If not, see
 ### <http://www.gnu.org/licenses/>.
 
 AC_PREREQ([2.63])
-AC_INIT([GNU Octave], [4.0.1], [http://octave.org/bugs.html], [octave])
+AC_INIT([GNU Octave], [4.0.2], [http://octave.org/bugs.html], [octave])
 
 dnl Note that the version number is duplicated here and in AC_INIT
 dnl because AC_INIT requires it to be static, not computed from
 dnl shell variables.
 OCTAVE_MAJOR_VERSION=4
 OCTAVE_MINOR_VERSION=0
-OCTAVE_PATCH_VERSION=1
+OCTAVE_PATCH_VERSION=2
 
 dnl PACKAGE_VERSION is set by the AC_INIT VERSION arg
 OCTAVE_VERSION="$PACKAGE_VERSION"
 
 OCTAVE_COPYRIGHT="Copyright (C) 2016 John W. Eaton and others."
 
-OCTAVE_RELEASE_DATE="2016-03-21"
+OCTAVE_RELEASE_DATE="2016-04-20"
 
 ## The "API version" is used as a way of checking that interfaces in the
 ## liboctave and libinterp libraries haven't changed in a backwardly
 ## incompatible way when loading .oct files.  A better way to do that is
 ## with library versioning, but not all systems support that.
 ## NOTE: This macro will be removed in a future version of Octave.  If
 ## you insist on checking for features using a version number, use the
 ## OCTAVE_MAJOR_VERSION, OCTAVE_MINOR_VERSION, and
diff --git a/libinterp/Makefile.am b/libinterp/Makefile.am
--- a/libinterp/Makefile.am
+++ b/libinterp/Makefile.am
@@ -195,17 +195,17 @@ liboctinterp_la_LIBADD = \
   parse-tree/libparser.la \
   corefcn/libcorefcn.la \
   corefcn/libtex_parser.la \
   $(top_builddir)/liboctave/liboctave.la \
   $(LIBOCTINTERP_LINK_DEPS)
 
 # Increment these as needed and according to the rules in the libtool manual:
 liboctinterp_current = 4
-liboctinterp_revision = 0
+liboctinterp_revision = 1
 liboctinterp_age = 1
 
 liboctinterp_version_info = $(liboctinterp_current):$(liboctinterp_revision):$(liboctinterp_age)
 
 liboctinterp_la_LDFLAGS = \
   -version-info $(liboctinterp_version_info) \
   $(NO_UNDEFINED_LDFLAG) \
   -bindir $(bindir) \
diff --git a/liboctave/Makefile.am b/liboctave/Makefile.am
--- a/liboctave/Makefile.am
+++ b/liboctave/Makefile.am
@@ -93,17 +93,17 @@ include link-deps.mk
 nodist_EXTRA_liboctave_la_SOURCES = dummy.cc
 
 liboctave_la_LIBADD += \
   $(top_builddir)/libgnu/libgnu.la \
   $(LIBOCTAVE_LINK_DEPS)
 
 # Increment these as needed and according to the rules in the libtool manual:
 liboctave_current = 3
-liboctave_revision = 1
+liboctave_revision = 2
 liboctave_age = 0
 
 liboctave_version_info = $(liboctave_current):$(liboctave_revision):$(liboctave_age)
 
 liboctave_la_LDFLAGS = \
   -version-info $(liboctave_version_info) \
   $(NO_UNDEFINED_LDFLAG) \
   @XTRA_CRUFT_SH_LDFLAGS@ \
